lib/http.js

```diff
@@ -142,16 +142,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     validate({query: 'bedrock-profile-http.account'}),
     asyncHandler(async (req, res) => {
       const {account: {id: accountId}} = (req.user || {});
-      const {account} = req.query;
+      const {profileAgentId} = req.params;
+      // this throws NotFoundError
+      const profileAgentRecord = await profileAgents.get({id: profileAgentId});
       // FIXME: Use http-sigs and zcaps for
       // TODO: Add permissions to allow access for admin
-      if(account !== accountId) {
+      if(profileAgentRecord.profileAgent.account !== accountId) {
```

Comment:
Ideally the remove call would take the account ID as well and include in the query so the removal is atomic. But, since this is probably a temporary solution anyway (with zcaps being the full solution) this non-atomic approach is better than what's there now.

Reply:
I added an issue for that: https://github.com/digitalbazaar/bedrock-profile/issues/65

Hope I got it right.